pkg/version/check_test.go

```diff
@@ -83,9 +76,9 @@ func (s *checkSuite) TestCheckClusterVersion(c *check.C) {
 		if err == nil {
 			break
 		}
-		c.Error(err)
+		fmt.Fprintf(os.Stderr, "[ERROR]:%v", err)
```

Comment:
How about we use `t.Log(f)`?

Also the original c.Error should make the case fail straight away.

Reply:
(1) t.Log(f) LGTM.
(2) c.Error will fail the test but not stop the test immediately.  So i can't find the corresponding function in  testify like c.Error.  testify has fail/failnow/failf function, but they will stop the test-case directly.    Any suggestions? @hi-rustin

Reply:
If it's all going to fail, then I see nothing wrong with ending it early. @overvenus What do you think?

Reply:
I find the testify.assert.Fail can act just like c.Error().  But the semantic of  assert in testify still confuses me, cause assert means severe mismatch in C/C++.

Reply:
😆 Cool!